client/service/src/metrics.rs

```diff
@@ -369,34 +174,23 @@ impl MetricsService {
 		);
 
 		if let Some(metrics) = self.metrics.as_ref() {
-			metrics.cpu_usage_percentage.set(process_info.cpu_usage as f64);
-			// `sysinfo::Process::memory` returns memory usage in KiB and not bytes.
-			metrics.memory_usage_bytes.set(process_info.memory * 1024);
-
-			if let Some(threads) = process_info.threads {
-				metrics.threads.set(threads);
-			}
-
-			if let Some(fd_info) = process_info.open_fd {
-				metrics.open_files.with_label_values(&["paths"]).set(fd_info.paths);
-				metrics.open_files.with_label_values(&["mem"]).set(fd_info.mem);
-				metrics.open_files.with_label_values(&["sockets"]).set(fd_info.sockets);
-				metrics.open_files.with_label_values(&["net"]).set(fd_info.net);
-				metrics.open_files.with_label_values(&["pipe"]).set(fd_info.pipes);
-				metrics.open_files.with_label_values(&["anon_inode"]).set(fd_info.anon_inode);
-				metrics.open_files.with_label_values(&["other"]).set(fd_info.other);
-			}
-
-
-			metrics.network_per_sec_bytes.with_label_values(&["download"]).set(
-				net_status.average_download_per_sec,
-			);
-			metrics.network_per_sec_bytes.with_label_values(&["upload"]).set(
-				net_status.average_upload_per_sec,
-			);
+			metrics
+				.network_per_sec_bytes
+				.with_label_values(&["download"])
+				.set(net_status.average_download_per_sec);
+			metrics
+				.network_per_sec_bytes
+				.with_label_values(&["upload"])
+				.set(net_status.average_upload_per_sec);
```

Comment:
Out of scope of this PR, but these are already reported by `sc-network` and are redundant here.